public/_headers

```diff
@@ -0,0 +1,17 @@
+https://oss-insights.netlify.app/*
+   X-Robots-Tag: noindex
+
+http://oss-insights.netlify.app/*
+   X-Robots-Tag: noindex
+
+https://*oss-insights.netlify.app/*
```

Comment:
Apparently Netlify include a `x-robots-tag: noindex` for deploy previews (but not for branch deploys). See https://answers.netlify.com/t/disabling-x-robots-tag-header-on-hosted-site/8857/8